code/game/turfs/open/lava.dm

```diff
@@ -175,3 +175,9 @@
 
 /turf/open/lava/smooth/airless
 	initial_gas_mix = AIRLESS_ATMOS
+
+/turf/open/lava/smooth/jungle
```

Comment:
It'll still be called lava, with the same description and it'll still set people on fire instead of just burning them. I'd call it boiling mud and describe it as a pool of bubbling slurry with steam coming off of it. 

> A mudpot, or mud pool, is a sort of acidic hot spring, or fumarole, with limited water. It usually takes the form of a pool of bubbling mud.

> The mud of a mudpot takes the form of a viscous, often bubbling, slurry.